docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+| Extension icon: active (full color), with a green tick                                                                                                                                                                                                                          | Extension icon: active (full color), but with a red X                           |
+| Opening the extension displays a **rate of pay** slider:<ul><li>On the left, the current hourly rate in the currency of the wallet.</li><li>The default is equivalent to 0.60 USD.</li><li>The remaining balance, updated in near real-time (i.e. every few seconds).</li></ul> | Opening the extension displays <ul><li>This website is not monetized.</li></ul> |
+
+##### Pay one-time when extension and wallet have enough funds
+
+**Test ID**: 3  
+ **Description**: Send a one-time payment to a monetized website when the remaining balance for the extension is sufficient for the payment, and the wallet has sufficient funds  
+ **Risk**: R3 (security of funds) and R6 (control my payments)  
+ **Preconditions**:
+
+1.  The extension is connected to your wallet.
+2.  The extension has a positive remaining balance.
+3.  Your wallet balance is equal to or greater than the extension’s remaining balance.
+
+**Steps**:
+
+1. Visit a monetized website. Refer to the [Test Data](#Test-Data) section to explore different WM conditions for websites.
+2. Open the extension and make a one-time payment of an amount that falls within your remaining balance shown in the extension.
+3. Visit a non-monetized website. Open the extension to observe its available options.
+
+**Expected results**:  
+ | Web monetized websites | Non monetized websites |
+| :---------------------------- | :--------------------------------------------------- |
+| Extension icon: active (full color), with a green tick | Extension icon: active (full color), but with a red X |
+| Opening the extension displays: <ul><li>**Rate of pay slider**: rate of pay and currency on the left, the remaining balance of the extension is on the right side, and it decreased by the value of the one-time payment</li><li>**Amount**: the one-time payment amount field resets to zero</li><li>**“Send now” button**: clicking the button to send a one-time payment changes the text to “Payment successful” for a few seconds, then the text defaults back to “Send now” </li></ul> | Opening the extension displays: <ul><li>This website is not monetized</li></ul> |
+
+##### Pay one-time when wallet is out of funds
+
+**Test ID**: 4  
+**Description**: Send a one-time payment to a monetized website when the extension has a sufficient remaining balance but the wallet has insufficient funds  
```

Comment:
I wonder how difficult it will be for testers to set up this scenario. Because they'd have to spend/move the existing funds in their wallet. Also, Test ID 5 wants the wallet to have sufficient funds, so maybe 5 should come before 4.

Reply:
Alrighty, both points addressed, thanks!